src/og/LonLat.js

```diff
@@ -20,31 +20,33 @@ const INV_PI_BY_180_HALF_PI = INV_PI_BY_180 * HALF_PI;
  * @param {number} [height] - Height over the surface.
  */
 export class LonLat {
+    static SW_MERC; // LatLng
+    static NE_MERC; // LatLng
+    static NW_MERC_DEG; // LatLng
+    static SE_MERC_DEG; // LatLng
+
+    /**
+     * Longitude.
+     */
+    lon = 0;

```

Comment:
in this case lon,lat, height are public.

Reply:
yes, they are public here. It's only if you prefix with `#` that they are private